README.md

```diff
@@ -507,9 +507,9 @@ zcbor validate --help
 ---------------------
 
 ```
-usage: zcbor validate [-h] [--version] -c CDDL [--no-prelude] [-v]
-                      [--default-max-qty DEFAULT_MAX_QTY] -i INPUT
+usage: zcbor validate [-h] [--version] -c CDDL [--no-prelude] [-v] -i INPUT
                       [--input-as {yaml,json,cbor,cborhex}] -t ENTRY_TYPE
+                      [--default-max-qty DEFAULT_MAX_QTY] [--sanitize]
```

Comment:
Why are we re-ordering existing arguments? The way this help text looks it could break users invoking this command in a given way (`-i` before `-v` is not required if I understand correctly).

Reply:
The reordering is just a consequence of some internal refactoring, it doesn't actually make a difference (these arguments can appear in any order after 'validate')